kie-internal/src/main/java/org/kie/internal/utils/ChainedProperties.java

```diff
@@ -155,6 +137,27 @@ public ChainedProperties(String confFileName,
                         this.defaultProps, confClassLoader );
         
         confClassLoader = Thread.currentThread().getContextClassLoader();
+        loadDefaultsFromClassLoader( confFileName, classLoader, confClassLoader );
+
+        try {
+            confClassLoader = ClassLoader.getSystemClassLoader();
+            loadDefaultsFromClassLoader( confFileName, classLoader, confClassLoader );
+        } catch (Throwable t) {
+            // DROOLS-1125: the system class loader cannot be retrieved in Google App Engine - ignore
+        }
+
+        // this happens only in OSGi: for some reason doing ClassLoader.getResources() doesn't work
+        // but doing Class.getResourse() does
+        if (this.defaultProps.isEmpty()) {
+            try {
+                Class<?> c = Class.forName("org.drools.compiler.lang.MVELDumper", false, classLoader);
+                URL confURL = c.getResource("/META-INF/drools.default." + confFileName);
+                loadProperties(confURL, this.defaultProps);
+            } catch (ClassNotFoundException e) { }
```

Comment:
Again the try catch needs to be on the small possible part to avoid eating exceptions it should not be eating.

Eating exceptions is incredable bad for a framework like drools: the unit tests don't detect it, but when something is wrong with users (regardless who's fault it is), then there is no fail fast, so the app misbehaves (and hopefully in a way that they notice immediately!) instead of showing an exception with a helpful error message.